### ISSUE TITLE
Some basic compile issue

### ISSUE THREAD

**Issue by zhaozhongch** [NVIDIA/libcudacxx#71]
I am really looking forward to using cxx in CUDA code. Thanks for the work. Still a newbie in this area so I have some basic questions.
My CUDA version is 10.2, which was installed months ago
1:
Do I need to install the libcudacxx separately (like `git clone` this repo, `cmake, make install`) or it is already in the cudatootkit?

2: Previously when I use CMake to compile CUDA code, it's something like the following
```
project(ABC LANGUAGES CXX CUDA)

find_package(CUDA QUIET REQUIRED)
include_directories(${CUDA_INCLUDE_DIRS} )

add_executable(abc abc.cu)
target_link_libraries(abc ${CUBLAS_LIBRARIES})
```
Which works fine.
I saw this repo's example cmake use 
```
find_package(CUDAToolkit REQUIRED)
```
I assume `find_package(CUDAToolkit REQUIRED)` take the place of `find_package(CUDA QUIET REQUIRED)`?
But when I use `find_package(CUDAToolkit REQUIRED)` in cmake I got 
```
...
  Could not find a package configuration file provided by "CUDAToolkit" with
  any of the following names:

    CUDAToolkitConfig.cmake
    cudatoolkit-config.cmake
...

```
Should I just upgrade my CUDA10.2 so that those things will work fine?

**Comment by griwes** [comment]
Unsure about the second question; @jrhemstad rewrote those cmakefiles so he's probably better to answer that one?

libcu++ is included in CUDA 10.2 and above; be mindful that the Debian packages for CUDA 10.2 are broken and have its headers in the wrong place.

You can also clone the repository from Github and point NVCC to the `include/` directory.

**Comment by zhaozhongch** [comment]
Thanks for the quick reply. I work on it this afternoon.
One thing I forgot to mention my system is Ubuntu18
Feels like when I installed CUDA 10.2 a long time ago so there is no libcuddacxx.
I reinstall CUDA11.1. and there is libcudaxcc. But seems the problem is `<cuda/std/atomic>` can only be used on windows since when I compile I got an error
```
#  error "CUDA atomics are only supported for sm_60 and up on *nix and sm_70 and up on Windows."
```
It will be great if some of these limits can be mentioned in the README.

About CMake compile, I need to add one more line 
```
include_directories(/usr/local/cuda/include)
```
so that the compiler can find things like `atomic` or `chrono`

And wait for something like `<cuda/std/vector>` can work. It seems from the example thrust vector is needed.
